app/src/main/java/fr/free/nrw/commons/upload/UploadClient.java

```diff
@@ -71,7 +71,7 @@ public Observable<StashUploadResult> uploadFileToStash(
             && contribution.getChunkInfo().getTotalChunks() == contribution.getChunkInfo()
             .getIndexOfNextChunkToUpload()) {
             return Observable.just(new StashUploadResult(StashUploadState.SUCCESS,
-                contribution.getChunkInfo().getUploadResult().getFilekey()));
+                contribution.getChunkInfo().getUploadResult().getFilekey(),"success"));
```

Comment:
Rather than passing "success", could you pass in `null` given that this value should just not be used in these cases?